src/test/java/internship/framework/pages/MyAccountPage.java

```diff
@@ -40,10 +40,10 @@ public boolean isValidateNewsletterDisplayed()
 
     }
 
-    public boolean checkValidateNewsletterText()
+    public String checkValidateNewsletterText()
```

Comment:
getValidateNewsletterText() would be a better naming

Reply:
Noted and fixed.